usage/installer/src/main/resources/brooklyn/entity/modaclouds/modaclouds.yaml

```diff
@@ -21,8 +22,5 @@ services:
     brooklyn.config:
       modaclouds.dda.ip: $brooklyn:component("dda").attributeWhenReady("host.address")
       modaclouds.dda.port: $brooklyn:component("dda").attributeWhenReady("modaclouds.dda.port")
-      modaclouds.kb.ip: $brooklyn:component("kb").attributeWhenReady("host.address")
-      modaclouds.kb.port: $brooklyn:component("kb").attributeWhenReady("modaclouds.kb.port")
-      modaclouds.kb.dataset.path: $brooklyn:component("kb").attributeWhenReady("modaclouds.kb.path")
```

Comment:
@mbarrientos The MODACLouds Knowledge Base is actually no more part of TOWER4 Clouds and it has been removed. It has been somehow replaced by the HistoryDB. Actually I did not set the link between the manager and the histoy db in the modaclouds.yaml since we are not sure about if the History DB will be used, but actually I put it in the modaclouds.yaml and so it is currently deployed. So as you suggest it requires some further configuration. Can I add this with another commit?